corehq/form_processor/migrations/0099_xforminstance_form_proces_domain_3bcfdd_idx_and_more.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 4.2.20 on 2025-10-14 14:13
+
+from django.db import migrations, models
+
+TABLE_NAME = 'form_processor_xforminstancesql'
+
+DOMAIN_STATE_INDEX_NAME = 'form_proces_domain_3bcfdd_idx'
+CREATE_DOMAIN_STATE_INDEX_SQL = "CREATE INDEX CONCURRENTLY IF NOT EXISTS {} ON {} ({}, {})".format(
+    DOMAIN_STATE_INDEX_NAME, TABLE_NAME, 'domain', 'state'
+)
+DROP_DOMAIN_STATE_INDEX_SQL = "DROP INDEX CONCURRENTLY IF EXISTS {}".format(DOMAIN_STATE_INDEX_NAME)
+
+DOMAIN_USER_INDEX_NAME = 'form_proces_domain_363a82_idx'
+CREATE_DOMAIN_USER_INDEX_SQL = "CREATE INDEX CONCURRENTLY IF NOT EXISTS {} ON {} ({}, {})".format(
+    DOMAIN_USER_INDEX_NAME, TABLE_NAME, 'domain', 'user_id'
+)
+DROP_DOMAIN_USER_INDEX_SQL = "DROP INDEX CONCURRENTLY IF EXISTS {}".format(DOMAIN_USER_INDEX_NAME)
```

Comment:
I think this technique of constructing CREATE/DROP statements with lots of constants, while succeeding at being DRY, is quite a bit harder to read than simply writing out the statements ([example](https://github.com/dimagi/commcare-hq/blob/10ff8ed4ca90a05369ac98e35ba89e67b7eb7eea/corehq/ex-submodules/phonelog/migrations/0016_rename_devicereportentry_domain_device_id_phonelog_da_domain_e98c65_idx_and_more.py)).


```suggestion
```